Tests/Amr/Advection_AmrCore/Source/AmrCoreAdv.cpp

```diff
@@ -161,16 +172,18 @@ AmrCoreAdv::InitData ()
         InitFromScratch(time);
         AverageDown();
 
+#ifdef AMREX_PARTICLES
+    init_particles();
```

Comment:
More consistent spaces:
```suggestion
        init_particles();
```